### PR TITLE
Adding three examples of automated experiments

### PR DESCRIPTION
**Pull request by gunnargrosch**
*Issue #, if available:*

Adding three examples of automated experiments:

- Scheduled
- Event triggered
- Continuous delivery

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
-*.js
```
[the remark this answers is not in the document]
Comment (jwoehrle) [reply .gitignore:1]: as an alternative add the .gitignore with *.js to the cdk subfolders

[the remark this answers is not in the document]
Comment (gunnargrosch) [reply .gitignore:1]: Sure thing. Did the first option.

**Review by jwoehrle — APPROVED** [review]
lgtm

**Comment by gunnargrosch** [comment]
@jwoehrle Can/will you do the merge?

**Pull request merged**